modules/@apostrophecms/area/index.js

```diff
@@ -43,9 +43,10 @@ module.exports = {
             throw self.apos.error('invalid');
           }
 
-          let options = field.options && field.options.widgets && field.options.widgets[type];
+          const widgets = this.getWidgets(field.options);
```

Comment:
This is random failing due to a typo `this.getWidgets` should become `self.getWidgets` (it happens in several places here)

Reply:
🤦‍♂️ fixed.